src/packages/src/xroad/common/proxy/usr/share/xroad/scripts/backup_db.sh

```diff
@@ -6,7 +6,7 @@ dump_file="$1"
 db_properties=/etc/xroad/db.properties
 db_host="127.0.0.1:5432"
 db_user="$(get_prop ${db_properties} 'serverconf.hibernate.connection.username' 'serverconf')"
-db_schema="$db_user"
+db_schema=serverconf
```

Comment:
hard-coded schema serverconf, but the db setup uses configurable schema name (same as username)